packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -94,7 +94,17 @@ export type CallControlOptions = {
 export const CallControls = (props: CallControlsProps): JSX.Element => {
   const { callInvitationURL, onFetchParticipantMenuItems } = props;
 
-  const options = typeof props.options === 'boolean' ? {} : props.options;
+  const options =
+    typeof props.options === 'boolean'
+      ? {
```

Comment:
This seems a little verbose for the call controls but for some reason there is two different behaviors here. When the prop is passed in from the meeting composite as just false it works fine, but when the meeting control bar passes in the same value of false on line 76 of this file the whole call controls bar shows up. Thoughts here as to what might be happening?